buildkite/bazelci.py

```diff
@@ -1087,7 +1115,10 @@ def test_logs_for_status(bep_file, status):
 
 def execute_command(args, shell=False, fail_if_nonzero=True):
     eprint(" ".join(args))
-    return subprocess.run(args, shell=shell, check=fail_if_nonzero, env=os.environ).returncode
+    process = subprocess.run(
+        args, shell=shell, check=fail_if_nonzero, env=os.environ, stdout=subprocess.PIPE
+    )
+    return process.stdout.decode("utf-8")
```

Comment:
Decoding with utf-8 might not be correct depending on the current locale. You can pass the weirdly named flag `universal_newlines=True` to [subprocess.run()](https://docs.python.org/3/library/subprocess.html#subprocess.run) and it will automatically handle decoding the stdout correctly. :)

Reply:
nice